.gitattributes

```diff
@@ -11,3 +11,7 @@ Makefile text
 *.bat eol=crlf
 *.txt text
 *.sh text
```

Comment:
just use
`*.sh eol=lf` - this implies `text` and it is reasonable to convert all shell scripts, not just travis